e3sm_diags/plot/cartopy/polar_plot.py

```diff
@@ -1,278 +0,0 @@
-from __future__ import print_function
-
-import os
-
-import cartopy.crs as ccrs
-import matplotlib
-import matplotlib.pyplot as plt
-import numpy as np
-import numpy.ma as ma
-
-from e3sm_diags.driver.utils.general import get_output_dir
-from e3sm_diags.logger import custom_logger
-from e3sm_diags.plot import get_colormap
-
-matplotlib.use("Agg")
-import matplotlib.colors as colors  # isort:skip  # noqa: E402
-import matplotlib.path as mpath  # isort:skip  # noqa: E402
-
-logger = custom_logger(__name__)
```

Comment:
Corresponds exactly in `e3sm_diags/plot/polar_plot.py` and `e3sm_diags/plot/utils.py`.